src/metadata/trackmetadata.h

```diff
@@ -1,5 +1,5 @@
-#ifndef TRACKMETADATA_H
-#define TRACKMETADATA_H
+#ifndef MIXXX_TRACKMETADATA_H
```

Comment:
Yes. And also because I've enclosed the class in namespace "Mixxx". I will check which headers are exposed to external plugins.